README.md

```diff
@@ -34,6 +34,7 @@ It's generally recommended not to use other cipher implementations in this repos
 | [SM4](https://en.wikipedia.org/wiki/SM4_(cipher)) | `sm4` | [![crates.io](https://img.shields.io/crates/v/sm4.svg)](https://crates.io/crates/sm4) | [![Documentation](https://docs.rs/sm4/badge.svg)](https://docs.rs/sm4) | ![MSRV 1.56][msrv-1.56] |
 | [Threefish](https://en.wikipedia.org/wiki/Threefish) | `threefish` | [![crates.io](https://img.shields.io/crates/v/threefish.svg)](https://crates.io/crates/threefish) | [![Documentation](https://docs.rs/threefish/badge.svg)](https://docs.rs/threefish) | ![MSRV 1.56][msrv-1.56] |
 | [Twofish](https://en.wikipedia.org/wiki/Twofish) | `twofish` | [![crates.io](https://img.shields.io/crates/v/twofish.svg)](https://crates.io/crates/twofish) | [![Documentation](https://docs.rs/twofish/badge.svg)](https://docs.rs/twofish) | ![MSRV 1.56][msrv-1.56] |
+| [Belt-Block](https://ru.wikipedia.org/wiki/BelT) | `belt-block` | [![crates.io](https://img.shields.io/crates/v/belt-block.svg)](https://crates.io/crates/belt-block) | [![Documentation](https://docs.rs/belt-block/badge.svg)](https://docs.rs/belt-block) | ![MSRV 1.56][msrv-1.56] |
```

Comment:
Please move it higher to keep the alphabetic order of crates.

Reply:
The belt also includes ciphers in another mode, CBC, ECB, CFB, also keywrap algo and others, how correct will such a name be?

Reply:
Specification in russian language, so you can see all algos included in this standard.
http://apmi.bsu.by/assets/files/std/belt-spec371.pdf

Reply:
Ah, I see. `belt-block` is used in the specification, so it indeed makes sense to call the cipher like that.

We have generic implementation of [block modes](https://github.com/RustCrypto/block-modes), which should be compatible with the specification (on the first glance I don't see any unnecessary weirdness like in the GOST). To ensure compatibility you can open PR against the block modes repository with added test vectors for BelT. `belt-mac` looks like a straightforward CMAC, but other algorithms may need new crates.

Reply:
All is done, thank you!